static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -24,8 +24,20 @@ export default class FilterVisibilityToggle extends SearchkitComponent {
     return checkFilterVisibility(filterName) ? "" : "closed";
   };
 
-  getChildFacetDocCount = (results: Object, resultId: string): number => {
-    const elementResult = _.get(results, ['aggregations', resultId]);
+  getChildFacetDocCount = (results: Object, resultIdPrefix: string): number => {
+    const aggregations = _.get(results, ['aggregations']);
+    let elementResult;
+    for (const id of Object.keys(aggregations)) {
+      // escape the periods in the key, then use that in a regex which matches exactly, plus maybe some digits
+      let regex = new RegExp(`^${resultIdPrefix.replace(".", "\\.")}(\\d*)$`);
+      if (regex.test(id)) {
+        elementResult = aggregations[id];
+        break;
+      }
+    }
+    if (!elementResult) {
+      return 0;
+    }
```

Comment:
i like the idea of just searching the aggregation keys for the prefix, but this regex approach is a bit clunky. here's another way to implement `getChildFacetDocCount`

```javascript
let matchingAggKey = R.compose(
  R.find(
    // Accept keys that start with the given prefix and end with numbers
    (k) => ( k.startsWith(resultIdPrefix) && !isNaN(k.substring(resultIdPrefix.length)) )
  ),
  R.keys
)(results.aggregations);

if (!matchingAggKey) {
  return 0;
}
else {
  let elementResult = results.aggregations[matchingAggKey];
  if (elementResult['inner']) {
    return elementResult['inner']['doc_count'];
  } else {
    return elementResult['doc_count'];
  }
}
```

you can also just do this to get the key and value rather than just the key: 
```javascript
let matchingAgg = R.pipe(
  R.toPairs,
  R.filter(R.apply(
  	(k, v) => (k.startsWith(resultIdPrefix) && !isNaN(k.substring(resultIdPrefix.length)))
  )),
  R.fromPairs
)(results.aggregations);
// matchingAgg would equal something like {"program.semester_enrollments.semester12": <agg_object>}
```